internal/cypress/config.go

```diff
@@ -28,9 +28,10 @@ type Project struct {
 	Defaults       config.Defaults `yaml:"defaults" json:"defaults"`
 	DryRun         bool            `yaml:"-" json:"-"`
 	ShowConsoleLog bool
-	ConfigFilePath string             `yaml:"-" json:"-"`
-	Sauce          config.SauceConfig `yaml:"sauce,omitempty" json:"sauce"`
-	Cypress        Cypress            `yaml:"cypress,omitempty" json:"cypress"`
+	ConfigFilePath string                 `yaml:"-" json:"-"`
+	CommandLine    map[string]interface{} `yaml:"-" json:"-"`
```

Comment:
Would be good if the name reflected the contents somewhat:

```suggestion
	Flags    map[string]interface{} `yaml:"-" json:"-"`
```

or

```suggestion
	CLIFlags    map[string]interface{} `yaml:"-" json:"-"`
```